packages/next/src/server-build.ts

```diff
@@ -313,7 +313,7 @@ export async function serverBuild({
 
     const useBundledServer =
       semver.gte(nextVersion, BUNDLED_SERVER_NEXT_VERSION) &&
-      process.env.VERCEL_NEXT_BUNDLED_SERVER;
+      process.env.VERCEL_NEXT_BUNDLED_SERVER === '1';
```

Comment:
Or you can coerce it into a boolean :)

```suggestion
      ~~process.env.VERCEL_NEXT_BUNDLED_SERVER;
```

Reply:
@cb1kenobi No we can't, see the PR description

Reply:
That's why I said `~~`:

```
> ~~null
0
> ~~undefined
0
> ~~''
0
> ~~false
0
> ~~0
0
> ~~'0'
0
> ~~true
1
> ~~1
1
> ~~'1'
1
```

Reply:
Too clever / unreadable IMO. I have no idea what that code does just by looking at it.

Reply:
Very strange, thats not even coercing to a boolean, its coercing to a number.

Almost the same as `Number(x)`. But I think we should be explicit here here so I'm going to leave it.

I agree with Nate

Reply:
I didn't even know `~` is an operator lol